client/lib/analytics/utils.js

```diff
@@ -7,46 +7,43 @@ function doNotTrack() {
 	return '1' === navigator.doNotTrack;
 }
 
+// If this list catches things that are not necessarily forbidden we're ok with
+// a little bit of approximation as long as we do catch the ones that we have to.
+// We need to be quite aggressive with how we filter candiate pages as failing
+// to protect our users' privacy puts us in breach of our own TOS and our
+// retargeting partners' TOS. We also see personally identifiable information in
+// unexpected places like email addresses in users' posts URLs and titles for
+// various (usually accidental) reasons. We also pass PII in URLs like
+// `wordpress.com/jetpack/connect` and `wordpress.com/error-report`.
+const forbiddenPiiPatterns = [
+	'@',
```

Comment:
I know `@` was on the original list, but that seems a bit aggressive? I guess if we've had no complaints previously, there's no need to change it.